be/src/storage/rowset_column_update_state.cpp

```diff
@@ -715,6 +727,13 @@ Status RowsetColumnUpdateState::finalize(Tablet* tablet, Rowset* rowset, uint32_
         cost_str << " [insert missing rows] " << watch.elapsed_time();
         watch.reset();
     }
+    // 5. release update chunk cache
+    std::for_each(_update_chunk_cache.begin(), _update_chunk_cache.end(), [&](auto& cache) {
+        if (!cache->is_empty()) {
+            tracker->release(cache->memory_usage());
+            cache = std::move(ChunkHelper::new_chunk(partial_schema, 0));
```

Comment:
Is it necessary to assign it to a new cache when finalize?

Reply:
Yes, it think so. Because we want to decrease memory in tracker when finalize, so we need to release this piece of memory.
We can't use `cache.reset(nullptr)` here, because we assume that will have a chunk ptr in `_update_chunk_cache`.
We can't use `cache->reset()` also, because it can't actual release the memory.
So what is your suggestion?

Reply:
Reset to null now.